bigtop-packages/src/common/ambari/patch9-remove-not-working-plugin-repos.diff

```diff
@@ -0,0 +1,70 @@
+diff --git a/ambari-server/pom.xml b/ambari-server/pom.xml
+index 8ff399051c..b268337ac8 100644
+--- a/ambari-server/pom.xml
++++ b/ambari-server/pom.xml
+@@ -1739,12 +1739,4 @@
+     </dependency>
+   </dependencies>
+
+-  <pluginRepositories>
+-    <pluginRepository>
+-      <id>oss.sonatype.org</id>
+-      <name>OSS Sonatype Staging</name>
+-      <url>https://oss.sonatype.org/content/groups/staging</url>
+-    </pluginRepository>
+-  </pluginRepositories>
+-
+ </project>
+diff --git a/ambari-serviceadvisor/pom.xml b/ambari-serviceadvisor/pom.xml
+index 8719cd85de..58d5dc749c 100644
+--- a/ambari-serviceadvisor/pom.xml
++++ b/ambari-serviceadvisor/pom.xml
+@@ -47,14 +47,6 @@
+     </dependency>
+   </dependencies>
+
+-  <pluginRepositories>
+-    <pluginRepository>
+-      <id>oss.sonatype.org</id>
+-      <name>OSS Sonatype Staging</name>
+-      <url>https://oss.sonatype.org/content/groups/staging</url>
+-    </pluginRepository>
+-  </pluginRepositories>
+-
```

Comment:
After applying this patch, references to oss.sonatype.org still seem to remain in ambari-infra/pom.xml and ambari-logsearch/pom.xml. Should we remove them (and the entry from the repos variable in build.gradle) too?

Reply:
Agreed, removed it!